addons/TBMod_main/functions/fn_animationAI.sqf

```diff
@@ -1,14 +1,32 @@
 ﻿/*
     Part of the TBMod ( https://github.com/shukari/TBMod )
     Developed by http://tacticalbacon.de
-    
-    Author: shukari
 */
 params ["_unit", "_value"];
 
-if (!alive _unit && _value != "") exitWith {};
+if (!alive _unit || _value == "") exitWith {};
 
 waitUntil {time > 0 && !isNil "TB_init_done"};
 
 [_unit, "ANIM"] remoteExec ["disableAI", _unit];
 [_unit, _value, 2] call ace_common_fnc_doAnimation;
+_unit setVariable ["TB_inAnim", true];
+
+TB_disableANIM = {
+    params ["_unit"];
+
+    // Selber kampffähig machen
+    if (_unit getVariable ["TB_inAnim", false]) then {
+        [_unit, "ANIM"] remoteExec ["enableAI", _unit];
+        [_unit, "", 1] call ace_common_fnc_doAnimation;
+        _unit setVariable ["TB_inAnim", false];
+    };
+    
+    {
+        if (_unit getVariable ["TB_inAnim", false]) then {[_x] call TB_disableANIM};
+    }
+    forEach (_unit nearEntities ["Man", 15]);
```

Comment:
Ist das ne Gute Idee? Eventhandler sind doch lokal, TB_inAnim ist aber lokal,ace_common_fnc_doAnimation ist global. Wird die funktion dadurch nicht mehrfach getriggert?

Reply:
Ähm nein.... TB_inAnim wird auf dem Server gesetzt... die Eventhandler werden auf dem Server ausgeführt... die Animation muss global sein damit sie jeder sieht... wo ist das Problem?

Reply:
Wir reden hier nicht über CBA Eventhandler... das sind normale BI Eventhandler.

Reply:
Mein Fehler. Sollte das nicht in ner Vorlesung reviewen.